delfin/drivers/utils/ssh_client.py

```diff
@@ -48,18 +48,28 @@ def connect(self):
         if self.ssh_private_key is None:
             self.ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
         else:
-            path = os.getcwd()
-            filename = path + '\\' + self.ssh_host + 'known_hosts'
-            file = open(filename, 'w')
-            file.write(self.ssh_private_key)
-            file.close()
-            self.ssh.load_host_keys(filename)
+            self.sethostkey(self.ssh, self.ssh_private_key)
 
         self.ssh.connect(hostname=self.ssh_host, port=self.ssh_port,
                          username=self.ssh_username,
                          password=self.ssh_password,
                          timeout=self.ssh_conn_timeout)
 
+    def sethostkey(self, ssh, line):
```

Comment:
Can you provide some comments here to expalin the purpose of this function? inputs, and return values..
What happens when it returns without setting self.ssh._host_keys ??

Reply:
sethostkey is used to set ssh public key,because input kwargs parameter 
    host_key is string, not a file path,we can not use load file to get pub
    lic key, parameter line is the public key which kwargs give us,this method
    have no return,if host key is none or if Set hostkey failed,it will Throw 
    out abnormal

Reply:
I mean please provide it as docstring in the start of function, so that developers will understnad what is this function meant for

Reply:
I have fixed it